src/ResourceManager/Network/Commands.Network/Properties/Resources.resx

```diff
@@ -339,4 +339,25 @@
   <data name="ApplicationGatewayTrustedRootCertificateName" xml:space="preserve">
     <value>trustedRootCertificates</value>
   </data>
+  <data name="P2SVpnServerConfigVpnClientRevokedCertificateName" xml:space="preserve">
```

Comment:
What are these being used for? These should mostly be strings that users will see.

Reply:
This is used to build the cert Id, its not visible to users.